plugins/SLURM-Integration/ufm_slurm_prolog.py

```diff
@@ -45,9 +45,14 @@ def prolog_init(self):
                         Constants.TOKEN_AUTH, Constants.CONF_TOKEN))
                     sys.exit(self.should_fail)
                 self.session = self.ufm.getServerSession(auth_type=self.auth_type, token=token)
+            elif self.auth_type == Constants.KERBEROS_AUTH:
```

Comment:
you are duplicating the logic from ufm_slurm_epilog - please define a function at UfmSlurmBase and reuse it...

Reply:
there is nothing that we can do her, because the epilog and prolog are two separated scripts, and for both of them we need to get principal name and password in order to get the session.

Reply:
fixes